docs/proposals/docs-process.md

```diff
@@ -0,0 +1,94 @@
+# Docs Process
+
+@johndmulhausen
+
+A new design for the kubernetes.io site is currently being implemented and we think it's a good time to streamline and simplify the docs creation process as we rebuild the site from scratch.
+
+When a community is empowered to be effective contributors, great things happen. That’s why I’m proposing that we move the documentation from the docs/ folder in the core Kubernetes repo in GitHub into the GitHub pages repo itself, migrate that repo into one named `kubernetes.github.io` so that it will be compatible with the GitHub Pages auto-update system, and leave all munging scripts behind.
+
+**TL;DR: Right now we do a lot of scripting and branching to create docs -- this proposal is about using Jekyll's native capability and GitHub Page's staging capability to create a much easier-to-edit set of documentation that will ship with our upcoming new site.**
+
+Benefits:
+
+* Onboard contributors quickly with tech they are familiar with that is easy to use
+* Have in-the-cloud staging of our site on `*username*.github.io`
+* Have documentation and website material located in one place
+* Reduce dependency on scripts and tools that only a few people can use/understand
+
+Cost:
+
+* Migration of current head/stable release system that is done with build/integration scripts into GitHub branches
+* Replacing of doc-munging scripts with native Jekyll functionality and/or GitHub capability.
+
+# Github.io staging
+
+GitHub has a technology called GitHub Pages that we currently use to construct the site at `kubernetes.io`. GitHub Pages is essentially a system for managing and hosting Jekyll sites. When you check in a Jekyll site into a repo that is named `username.github.io`, then GitHub will automatically stage its contents at `http://username.github.io`, and automatically update it whenever you push new changes -- no publishing required.
+
+This highly lubricates the ability for external users to make changes, as they can immediately see the consequences of their actions in their forked environment. 
+
+## Why migrate to kubernetes.github.io instead of using a `gh-pages` branch?
+
+GitHub Pages also suppots using branches of an existing repo to host Jekyll sites; in this case you would reuse the repo, but check in the Jekyll site to a branch named `gh-pages`. While you still get auto-building and staging, it will build at `http://username.github.io/**repo**` -- and in our case that is problematic because absolute links to any URI within the site will not work the same as they will on kubernetes.io. If we use the `username.github.io` repo instead of the gh-pages branch, then including an image from `/images/example.png` will work the same on `http://kubernetes.github.io`, `http://your_username.github.io`, `http://kubernetes.io`, and your local machine (running `jekyll serve` off of a clone of our repo).
```

Comment:
Devil's advocate: you can still make gh-pages show up as the root of your personal repo, you just have to carry a single commit to _config.yaml.